src/printer.rs

```diff
@@ -295,44 +324,40 @@ impl<W: WriteColor> Printer<W> {
                 self.write_eol();
                 return;
             }
-            self.write(&line);
-            if line.last() != Some(&self.eol) {
-                self.write_eol();
-            }
+            self.write_matched_line(offsets, &line[..], false);
```

Comment:
I think `&line` should be sufficient here? I guess you might need `&*line` because `line` is a cow I think.

Reply:
Yeah, `&*line` is needed.